cristin-person/src/main/java/no/unit/nva/cristin/person/model/cristin/CristinPerson.java

```diff
@@ -388,6 +391,21 @@ private Set<Employment> extractEmployments() {
                    .collect(Collectors.toSet());
     }
 
+    private Boolean extractVerified() {
+        var nviVerified = Optional.ofNullable(extractNvi())
+                              .map(PersonNvi::verifiedBy)
+                              .map(PersonSummary::id)
+                              .map(UriUtils::extractLastPathElement)
+                              .filter(Utils::isPositiveInteger)
+                              .isPresent();
+
+        if (nviVerified) {
+            return true;
+        } else {
+            return getIdentifiedCristinPerson();
```

Comment:
Would it be less "expensive" to check `getIdentifiedCristinPerson()` first?

Reply:
Yes, i will change the order

Reply:
Updated method